### ISSUE TITLE
it repository is deprecated?

### ISSUE THREAD

**Issue by enieber** [cnjon/react-native-pdf-view#77]
it repository is not update to npm since 24/04.
have issues like #60 and #66 that make one workaround to work in ios.
have outher repositories on npm that fork this repository. one sample is https://www.npmjs.com/package/@cqingwang/react-native-pdf-view

**Comment by mkoo21** [comment]
Yes the owner of this repo gave collab access to very few people and has been MIA for a long time, making it very difficult to maintain this repo. 

Please use the forked lib.

**Comment by wonday** [comment]
You can use the new one:
https://github.com/wonday/react-native-pdf
